### PR TITLE
Neos service listing

### PR DESCRIPTION
**Pull request by davidspiola**

### CODE REVIEW

File: DistributionPackages/Neos.NeosIo.ServiceOfferings/Configuration/NodeTypes.Mixin.Contributions.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    packages:
+      type: references
+      ui:
+        label: 'Packages'
+        reloadIfChanged: true
+        inspector:
+          group: contributions
+          editorOptions:
+            nodeTypes:
+              - 'Neos.MarketPlace:Vendor'
```
Comment (regniets) [inline DistributionPackages/Neos.NeosIo.ServiceOfferings/Configuration/NodeTypes.Mixin.Contributions.yaml]: Change to `Neos.MarketPlace:Package`

File: DistributionPackages/Neos.NeosIo.ServiceOfferings/Resources/Private/Fusion/Documents/ServiceProvider.fusion [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                                <Neos.Fusion:Loop items={props.caseStudies} itemName="caseStudy">
+                                    <li>{caseStudy.title}</li>
+                                </Neos.Fusion:Loop>
+                            </ul>
+                        </div>
+
+                        <div @if.set={props.packages}>
+                            <h2>Packages</h2>
+                            <ul>
+                                <Neos.Fusion:Loop items={props.packages} itemName="package">
```
Comment (regniets) [inline DistributionPackages/Neos.NeosIo.ServiceOfferings/Resources/Private/Fusion/Documents/ServiceProvider.fusion]: This should be changed if we leave the NodeType-Constraint with ´Neos.MarketPlace:Vendor´

**Comment by Sebobo** [comment]
@regniets good point. I think we actually more options.

There are people/agencies with several vendor names and there are people like me who maintain packages in other vendors namespaces (Yoast).

**Comment by regniets** [comment]
We could make it Vendors and Packages in NodeTypes Def and get the Vendors' children in fusion then to have only packages to show.

**Comment by Sebobo** [comment]
Yes that's also what I thought of. Only needs some minor adjustments then.

**Comment by Sebobo** [comment]
@regniets  
I made the discussed adjustments with every provider having multiple packages and multiple vendor names:
Bildschirmfoto 2019-11-06 um 14 02 48

**Comment by regniets** [comment]
@Sebobo - great, thanks!

**Comment by Sebobo** [comment]
I will now just merge this with the map so we can discuss it tomorrow

**Pull request merged**